### DISCUSSION TITLE
Storage | System Design

### DISCUSSION THREAD

**Discussion by giscus** [karanpratapsingh/portfolio#41] [Comments]
# Storage | System Design

In this tutorial, we will discuss Storage

https://www.karanpratapsingh.com/courses/system-design/storage

**Comment by Leader-board** [comment]
I don't fully agree with the table, specifically the Cost part. 

If "RAID 0" is considered as low-cost, "RAID 1" must also be low-cost, since the cost is the same (two drives) - with RAID 5 onwards being medium/high due to the increased number of drives required.

**Comment by bfdes** [comment]
The capacity utilization of RAID 6 is 94% if you use the maximum number of disks it supports, which is 32.
